src/code_809A560.c

```diff
@@ -237,29 +237,32 @@ void sub_809A7EC(void)
     }
 }
 
-// https://decomp.me/scratch/4MpWc (96.23% matching - Seth)
-// void sub_809A83C(s16 index)
-// {
-//     struct unkStruct_3001B64_sub *temp;
-//
-//     temp = &gUnknown_3001B64->unk43C[index];
-//
-//     temp->unk0 = -1;
-//     temp->speciesID = MONSTER_NONE;
-//     temp->unk4 = 0;
-//     temp->unk5 = 0;
-//     temp->unk6 = -1;
-//     temp->unk7 = 0;
-//     temp->unk8 = 0;
-//     temp->unkC = 0;
-//     temp->unk10 = 0;
-//     temp->faceData = NULL;
-//     temp->unk1C = 0;
-//     strcpy(gAvailablePokemonNames + (index * 0x50), gInvalidityText);
-//     strcpy(gPlayerName + (index * 0x50), gInvalidityText);
-//     if(temp->faceFile)
-//     {
-//         CloseFile(temp->faceFile);
-//         temp->faceFile = NULL;
-//     }
-// }
+static inline struct unkStruct_3001B64_sub *LOL(s16 p, struct unkStruct_3001B64 **thing) {
```

Comment:
We should probably use a temp name that makes it more clear that this is an undocumented function.

Reply:
`sub_809A83C_inline`?

Reply:
Yeah, that sounds good.